src/encoded/static/components/objectutils.js

```diff
@@ -870,6 +870,20 @@ export const filterForDefaultFiles = (fileList) => (
 );
 
 
+/**
+ * Filter the given files to only include those that appear in the given datasets.
+ * @param {array} fileList Files to filter
+ *
```

Comment:
Why the gap?

Reply:
There was something I saw that seemed authoritative that there should be a gap before @returns, but now I don’t see it and wonder if I imagined it. I’ll not do that anymore.